index.html

```diff
@@ -100,6 +100,122 @@ <h2 id="connect">Let's Connect</h2>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <div id="button-container">
+            <button type="button" class="btn">
+              <a href="#">See Project</a>
+            </button>
+          </div>
+        </li>
+        <li class="achomplishment-card">
+          <img
+            src="images/Second-work.png"
+            alt="work_multi-post_stories_daily_personalized_reads"
+          />
+          <h2>Multi-Post Stories</h2>
+          <ul class="skill">
+            <li>Canopy</li>
+            <li>&#x2022;</li>
+            <li>Back End Dev</li>
+            <li>&#x2022;</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="technologies">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <div id="button-container">
+            <button type="button" class="btn">
+              <a href="#">See Project</a>
+            </button>
+          </div>
+        </li>
+        <li class="achomplishment-card">
+          <img
+            src="images/third-work.png"
+            alt="work_multi-post_stories_daily_personalized_reads"
+          />
+          <h2>Tonic</h2>
+          <ul class="skill">
+            <li>Canopy</li>
+            <li>&#x2022;</li>
+            <li>Back End Dev</li>
+            <li>&#x2022;</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="technologies">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <div id="button-container">
+            <button type="button" class="btn">
+              <a href="#">See Project</a>
+            </button>
+          </div>
+        </li>
+        <li class="achomplishment-card">
+          <img
+            src="images/fourth-work.png"
+            alt="work_multi-post_stories_daily_personalized_reads"
+          />
+          <h2>Multi-Post Stories</h2>
+          <ul class="skill">
+            <li>Canopy</li>
+            <li>&#x2022;</li>
+            <li>Back End Dev</li>
+            <li>&#x2022;</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="technologies">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <div id="button-container">
+            <button type="button" class="btn">
+              <a href="#">See Project</a>
+            </button>
+          </div>
+        </li>
+      </ul>
+  </section>
   </body>
 </html>
```

Comment:
- [ ] You made great work on this project but I'm not able to see your  about page , kindly take care of this since your design is not yet complteted :wink: 
 I can't see this on your project :upside_down_face: 
![aboutsection](https://user-images.githubusercontent.com/88894525/205457681-57e96ffb-7328-4dbd-be76-7a26f69e40a1.png)

Reply:
I think it is visible in my view, I work it clearly by sticking on the template, please try to see the drop down link.

Reply:
- Hello @fithamlak, just like the previous reviewer mentioned, the `about-myself` section seems to be missing on your website 😢 , I have cloned your project locally and it is still missing: Please ensure that you have pushed all your latest changes to your repo 👍 because I cannot see the HTML section responsible for displaying the said section.

HTML code from your repo:
<img width="898" alt="ryrykry" src="https://user-images.githubusercontent.com/73607512/205459320-a416d581-bbe5-4537-b29b-420dc44565f4.PNG">